src/meta/src/manager/catalog/mod.rs

```diff
@@ -1240,9 +1351,44 @@ impl CatalogManager {
             }
         }
 
+        // FIXME(kwannoel): Propagate version to fe
+        let _version = self
```

Comment:
See comment on changes in frontend `drop_mv`.

Reply:
https://github.com/risingwavelabs/risingwave/pull/17484#discussion_r1663903029

Reply:
Requires a larger refactor to propagate notification version to frontend as well. Because the deleting catalog step happens within `create_streaming_job_inner`, when it receives a cancel notification. So it does not occur directly inside the path of the `cancel request`, making it hard to propagate currently.